mediator/med_phases_prep_glc_mod.F90

```diff
@@ -892,36 +894,26 @@ subroutine med_phases_prep_glc_renormalize_smb(gcomp, rc)
     call ESMF_GridCompGetInternalState(gcomp, is_local, rc)
     if (chkErr(rc,__LINE__,u_FILE_u)) return
 
-    !---------------------------------------
-    ! Get vm
-    !---------------------------------------
-
-    call ESMF_GridCompGet(gcomp, vm=vm, rc=rc)
-    if (ChkErr(rc,__LINE__,u_FILE_u)) return
-
     !---------------------------------------
     ! Map Sg_icemask_g from the glc grid to the land grid.
     !---------------------------------------
 
     ! determine Sg_icemask_g and set as contents of FBglc_icemask
-    call FB_getFldPtr(is_local%wrap%FBImp(compglc,compglc), trim(Sg_icemask_field), fldptr1=dataptr1d, rc=rc)
+    call ESMF_FieldBundleGet(is_local%wrap%FBImp(compglc,compglc), fieldname=trim(Sg_icemask_fieldname), field=lfield, rc=rc)
+    if (chkerr(rc,__LINE__,u_FILE_u)) return
+    call ESMF_FieldGet(lfield, farrayptr=dataptr1d, rc=rc)
     if (chkerr(rc,__LINE__,u_FILE_u)) return
-    call FB_getFldPtr(FBglc_icemask, trim(Sg_icemask_field), fldptr1=Sg_icemask_g,  rc=rc)
+    call ESMF_FieldGet(field_glc_icemask_g, farrayptr=Sg_icemask_g, rc=rc)
     if (chkerr(rc,__LINE__,u_FILE_u)) return
     Sg_icemask_g(:) = dataptr1d(:)
 
-    ! BUG(wjs, 2017-05-11, #1516) I think we actually want norm = .false. here, but this
-    ! requires some more thought
-    call med_map_FB_Regrid_Norm( &
-         fldsSrc=fldList_glc2lnd_icemask%flds, &
-         srccomp=compglc, &
-         destcomp=complnd, &
-         FBSrc=FBglc_icemask, &
-         FBDst=FBlnd_icemask, &
-         FBFracSrc=FBglc_icemask, &  ! this will not be used since are asking for 'none' normalization
-         FBNormOne=is_local%wrap%FBNormOne(compglc,complnd,:), &
-         RouteHandles=is_local%wrap%RH(compglc,complnd,:), &
-         string='mapping Sg_imask_g to Sg_imask_l (from glc to land)', rc=rc)
+    ! map ice mask from glc to lnd with no normalization
+    ! BUG(wjs, 2017-05-11, #1516) I think we actually want norm = .false. here, but this needs more thought
```

Comment:
Is this comment still valid?  Don't see the referenced 'norm'

Reply:
I've added a clarifying statement.